modules/statics/src/ada.ts

```diff
@@ -0,0 +1,80 @@
+import { BaseCoin, CoinFeature, CoinKind, KeyCurve, UnderlyingAsset } from './base';
+import { AdaNetwork } from './networks';
+
+export interface AdaConstructorOptions {
+  fullName: string;
+  name: string;
+  network: AdaNetwork;
+  features: CoinFeature[];
+  asset: UnderlyingAsset;
+  prefix?: string;
+  suffix?: string;
+  primaryKeyCurve: KeyCurve;
+}
+
+export class Ada extends BaseCoin {
+  public static readonly DEFAULT_FEATURES = [
+    CoinFeature.UNSPENT_MODEL,
```

Comment:
Question about this - we make distinctions between bitcoin-like unspent models and nonbitcoin-like unspent models in other places like bitgo-microservices, is there a point to do it here or no?

Reply:
Not 100% sure, I'm not familiar with what it looks like in bitgo-microservices. Could you link to the distinction so I could get some context?